crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -75,6 +75,10 @@ impl<T: Pod> BufferVec<T> {
         index
     }
 
+    fn append(&mut self, other: &mut BufferVec<T>) {
```

Comment:
```suggestion
    pub fn append(&mut self, other: &mut BufferVec<T>) {
```
Could you make it public?

Reply:
Good catch! That also seems to have resolved the dead code error during build